flume/neore/rpm/SPECS/cygnus.spec

```diff
@@ -36,6 +36,9 @@ a Flume-based connector for context data coming from Orion Context Broker.
 %define _project_user cygnus
```

Comment:
Should the fix of issues #329 and #330 be mentioned in the CHANGES_NEXT_RELEASE file?